frontend/src/containers/control/add/index.js

```diff
@@ -249,14 +256,32 @@ export default class ControlView extends React.Component {
     }
   }
 
-
+  oneStateIsNotOK() {
+    return (this.state.temperatureOK === false ||
+      this.state.relativeHumidityOK === false ||
+      this.state.inertAirOK === false ||
+      this.state.lightConditionsOK === false ||
+      this.state.cleaningOK === false ||
+      this.state.gasOK === false ||
+      this.state.alchoholOK === false ||
+      this.state.moldFungusOK === false ||
+      this.state.pestOK === false)
+  }
+  onClickSave() {
+    if (this.oneStateIsNotOK()) {
+      this.props.onLagreControl(this.state)
+      hashHistory.push('/observation/control/add/3')
```

Comment:
The id should not be dummy. If nothing is created at this point adjust the route by removing the ID.